src/main/java/seedu/address/logic/commands/ClientStatusCommand.java

```diff
@@ -95,7 +95,8 @@ public CommandResult execute(Model model) throws CommandException {
     }
 
     /**
-     * Generates a command execution success message based on whether the policy is added to or removed from
+     * Generates a command execution success message based on whether the {@code ClientStatus} is upgraded, downgraded,
+     * or reset.
      * {@code personToEdit}.
      */
```

Comment:
Great job on the JavaDoc! It clearly explains the purpose of the method and the conditions under which different success messages are generated. The use of {@code ClientStatus} to refer to the class is also a good practice for clarity.